content/6_8_plug-process-other-loads.html

```diff
@@ -357,27 +436,36 @@ <h5>90.1 2019 and 2022 PRM</h5>
                     <p>The schedules must be typical of the proposed building type as determined by the designer and approved by the rating authority.</p>
                     <p><strong>Table G3.1 #12</strong></p>
                     <ul>
-                        <li>Receptacle schedules must be the same as the proposed design before the receptacle power credit is applied.
+                        <li>Receptacle schedules must be the same as the proposed design before the receptacle power credit is applied.</li>
```

Comment:
@Priya-N1 You are noticeably getting better at this! This whole file only had one syntax mistake, otherwise I did not do anything.